tests/unit/certGen.unitTest.ts

```diff
@@ -42,6 +42,127 @@ describe("cert-gen", () => {
   context("CertificateGenerationService", () => {
     LambdaMockService.populateFunctions();
 
+    context("when a passing test result is read from the queue", () => {
+      const event: any = { ...queueEventPass };
+      const testResult: any = JSON.parse(event.Records[3].body);
+      const testResult2: any = JSON.parse(event.Records[4].body);
+
+      context("and a payload is generated", () => {
+        context("and no signatures were found in the bucket", () => {
+          it("should return a VTP20 payload without signature", () => {
+            const expectedResult: any = {
+              Watermark: "NOT VALID",
+              DATA: {
+                TestNumber: "W01A00310",
+                TestStationPNumber: "09-4129632",
+                TestStationName: "Abshire-Kub",
+                CurrentOdometer: {
+                  value: 12312,
+                  unit: "kilometres",
+                },
+                IssuersName: "Dev1, CVS",
```

Comment:
I think there are ACs (3 & 4?) which are not covered, I believe we have 4 scenarios but we have 2 unit tests. I would need a catch up on AC1 & 2 please when you have time.
We could use this jest tables to test the permutations: https://jestjs.io/docs/26.x/api#2-testeachtablename-fn-timeout